src/metabase/public_settings.clj

```diff
@@ -71,6 +71,25 @@
   :audit   :getter
   :default true)
 
+(defn- set-update-channel! [new-channel]
+  (let [valid-channels ["latest" "beta" "nightly"]]
+    ;; check to make sure `update-channel` is a valid channel, or throw an Exception it is it not.
+    (when-not (some #{new-channel} valid-channels)
```

Comment:
```suggestion
  (let [valid-channels #{"latest" "beta" "nightly"}]
    (when-not (valid-channels new-channel)
```

not a high-conviction change, but saves a couple characters and probably a couple clock cycles

Reply:
(and I don't think the comment is helpful)